ext/curl/tests/curl_setopt_CURLOPT_PREREQFUNCTION.phpt

```diff
@@ -0,0 +1,61 @@
+--TEST--
+Curl option CURLOPT_PREREQFUNCTION
+--EXTENSIONS--
+curl
+--SKIPIF--
+<?php
+$curl_version = curl_version();
+if ($curl_version['version_number'] < 0x075000) die("skip: test works only with curl >= 7.80.0");
+?>
+--FILE--
+<?php
+include 'server.inc';
+
+$host = curl_cli_server_start();
+$port = (int) (explode(':', $host))[1];
+
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL, $host);
+curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+//curl_setopt($ch, CURLOPT_IPRESOLVE, CURL_IPRESOLVE_V4);
```

Comment:
Why is this commented?

Reply:
I think I was trying to fix the cli server connections were using IPv6 loopback address, so my previous test that expected an IPv4 address failed. I replaced with a `filter_var` / `FILTER_VALIDATE_IP` call, so it wasn't needed. I cleaned it up.